test/browser/test/failing.test.ts

```diff
@@ -21,6 +21,16 @@ it('several locator methods are not awaited', () => {
   page.getByRole('button').tripleClick()
 })
 
+describe.runIf(server.provider === 'playwright')('timeouts are failing correctly', () => {
+  it('click on non-existing element fails', async () => {
+    await page.getByRole('code').click()
```

Comment:
To exercise remaining time subtraction logic, can we add a random sleep here?

```suggestion
    await new Promise(r => setTimeout(r, 500))
    await page.getByRole('code').click()
```